### PR TITLE
resurrecting PR #20

### PR DESCRIPTION
**Pull request by thatandromeda**
#20 has been languishing for years and should be resolved prior to conference. It appears to contain:
- some uncontroversial wording changes which are consistent with the current wording of the document;
- other uncontroversial wording changes which are now obsolete;
- a controversial change to remove references to specific sanctions.

This PR applies the first of these, and does not apply the second two. If @dltj , @kcoyle, or anyone else would like to remove reference to specific sanctions, please open a separate PR that is limited in scope to that one change and make your case in comments - we are clearly going to need more discussion to reach consensus.

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -113,21 +112,20 @@ provide escorts, or otherwise assist those experiencing harassment to feel
...
+In the case of organized events, those responsible for the event should have readily available the following information:
 
 * **Conference organizer:** [ORGANIZER NAME], [PHONE NUMBER]
 * **[CONFERENCE HOTEL NAME]:** [PHONE NUMBER]
 * **[MUNICIPALITY] Police Department**: [PHONE NUMBER]
 * **[LOCAL SEXUAL ASSAULT/CRISIS LINE]:** [PHONE NUMBER]
 * **[TAXI COMPANY NAME]:** [PHONE NUMBER]
-* **IRC channel administrators:** [IRC USERNAMES] (TODO: reference to
-    `@helpers`?)
+* **IRC channel administrators:** [IRC USERNAMES]
```
Comment (anarchivist) [inline code_of_conduct.md]: IIRC I think there was an issue about the semantics of "administrators". Does this include:

* people with admin access to `zoia`, who I believe can invoke the bot's `kick`, `ban`, etc. commands?
* people who are [Freenode "group contacts"](http://freenode.net/faq.shtml#contents-groups) for #code4lib? I think the only person who may be identified as such is Todd Holbrook, who isn't a regular on #code4lib anymore.

**Comment by anarchivist** [comment]
I would like to clarify the "IRC channel administrators" issue, but I'm otherwise :+1: on this.

This repository has been active recently; is it worth making the main list (or the conference list) aware of these changes?

**Comment by kcoyle** [comment]
I don't recall what I was thinking before. Today the list of sanctions looks more to me like guidance for organizers, and kind of has their back if they make one of those decisions. The other wording fixes look ok to me. The one that might be controversial is the change from "threatens" to "demeans" -- I think this speaks to the difference between harassment and discrimination, and I don't know how we want to deal with this. I'd be fine with a combined 'antiharassment/anti-discrimination' code. I think that antiharassment speaks more to sexual content, while anti-discrimination covers race, religion, etc. Presumably we wish to address the latter as well.

**Comment by yo-bj** [comment]
Do you think a change to include both words would help? Possibility: "threatens and/or demeans" to replace "threatens"?

**Comment by kcoyle** [comment]
I'd happy go with "threatens or demeans" - just skipping the and/or

**Comment by anarchivist** [comment]
:+1: to @kcoyle's suggestion.

**Comment by yo-bj** [comment]
+1

**Comment by dltj** [comment]
+1 to @kcoyle's suggestion.

**Comment by yo-bj** [comment]
I move to get this merged with amendments to the specific lines pointed out in the discussion.

**Comment by thatandromeda** [comment]
I have amended the PR to address feedback and emailed c4l to invite them to have a look.

There are some event-specific contacts that the c4l15 organizers need to fill in, and I don't see an obvious link on the conference web site; can someone who is involved with conference organizing herd those cats?  Also someone who is not me should merge the PR if there are no further issues.

File: code_of_conduct.md [hunk 2]
```diff
@@ -5,21 +5,20 @@ CodeOfConduct4Lib
 
 Short link: <http://bit.ly/coc4lib>
 
-Code4Lib seeks to provide a welcoming, fun, and safe community and
-conference experience and ongoing community for everyone. We do not
-tolerate harassment in any form. Discriminatory language and imagery
+Code4Lib seeks to provide a welcoming, professionally engaging, fun, and
+safe conference experience and ongoing community for everyone. We do not tolerate harassment in any form. Discriminatory language and imagery
```
Comment (jcoyne) [inline code_of_conduct.md]: All the other lines appear to be formatted to 80 chars max. Do you want to break the line prior to `tolerate`?

File: code_of_conduct.md [hunk 3]
```diff
@@ -5,21 +5,20 @@ CodeOfConduct4Lib
...
+Harassment is understood as any behavior that threatens or demeans another
+person or group, or produces an unsafe environment.  It includes offensive
+verbal comments or non-verbal expressions related to gender, gender identity,
 gender expression, sexual orientation, disability, physical appearance,
-body size, race, age, religious beliefs, sexual or discriminatory images
-in public spaces (including online), deliberate intimidation, stalking,
-following, harassing photography or recording, sustained disruption of 
-talks or other events, inappropriate physical contact, and unwelcome sexual
-attention.
+body size, race, age, religious or political beliefs; sexual or
```
Comment (jcoyne) [inline code_of_conduct.md]: Many political beliefs are offensive and discriminatory. Would this muddle the original intent?

File: code_of_conduct.md [hunk 4]
```diff
@@ -113,21 +112,20 @@ provide escorts, or otherwise assist those experiencing harassment to feel
...
+In the case of organized events, those responsible for the event should have readily available the following information:
 
 * **Conference organizer:** [ORGANIZER NAME], [PHONE NUMBER]
 * **[CONFERENCE HOTEL NAME]:** [PHONE NUMBER]
 * **[MUNICIPALITY] Police Department**: [PHONE NUMBER]
 * **[LOCAL SEXUAL ASSAULT/CRISIS LINE]:** [PHONE NUMBER]
 * **[TAXI COMPANY NAME]:** [PHONE NUMBER]
-* **IRC channel administrators:** [IRC USERNAMES] (TODO: reference to
-    `@helpers`?)
+* **zoia administrators:** [IRC USERNAMES]
```
Comment (jcoyne) [inline code_of_conduct.md]: What does Zoia have to do with this? Isn't a IRC channel operator/helper really what is necessary?

File: code_of_conduct.md (hunk 3 above)
Comment (jcoyne) [inline code_of_conduct.md]: Are there any other known codes of conduct that specifically include political beliefs? I'm not finding any other examples.

File: code_of_conduct.md (hunk 4 above)
Comment (anarchivist) [inline code_of_conduct.md]: @jcoyne This change (I'm guessing) has to do with my [comment](https://github.com/code4lib/antiharassment-policy/pull/50#discussion_r23354153) on the ambiguity of the phrase "IRC channel adminstrator" in the context of #code4lib.

Comment (gmcharlt) [inline code_of_conduct.md]: I agree -- either "IRC channel administrators" or referring to @helpers seems better; the bot is not the channel.

File: code_of_conduct.md (hunk 3 above)
Comment (gmcharlt) [inline code_of_conduct.md]: Here's one: http://www.composeconference.org/conduct/

As a counterpoint, http://www1.umn.edu/ohr/policies/governing/civilrules/rule2/ is a policy that prohibits discrimination on the basis of political opinions and affiliation.

I think "political affiliation" would be better for the policy, perhaps in conjunction with a guideline that attendees should immediate withdraw from a conversation if asked by the other party.

**Comment by anarchivist** [comment]
Resurfacing this pull request. I have a [branch on my fork](https://github.com/anarchivist/antiharassment-policy/tree/pr-50) that addresses the line-length issue and replaces the reference to the Zoia admins with more explicit instructions on how to use `@helpers` publicly or via a private message. [Compare here](https://github.com/thatandromeda/antiharassment-policy/compare/master...anarchivist:pr-50).

I did not change `political beliefs` to `political affiliation`. 

@thatandromeda - feel free to grab that commit if you'd like to add it to this branch, or let me know if it merits further discussion. Like you, I'm hoping we can get this in before the 2016 conference (even though this was opened before the 2015 conference!).

**Comment by jcoyne** [comment]
:+1:

**Comment by anarchivist** [comment]
Bump? It's been over two years since this PR was opened.

**Comment by escowles** [comment]
I am 👍  on merging this, with @anarchivist's changes (https://github.com/code4lib/antiharassment-policy/pull/50#issuecomment-171154849)

**Comment by anarchivist** [comment]
I reckon we should tag someone on the LPC too, since it'll probably impact the website. @joshuago78 @highermath ping?

**Comment by gmcharlt** [comment]
+1 for merging @anarchivist's commit bb515f8e7e0025d45

**Comment by gmcharlt** [comment]
Per discussion today during the duty officer's meeting, @anarchivist and I are not planning on merging commit bb515f8 or any other change to the code of conduct right before the conference is to begin. This pull request really ought to be resolved before the next conference, however.

**Comment by yo-bj** [comment]
Bump - is it time to revisit this PR and decide if we will accept the changes or not?

**Comment by anarchivist** [comment]
I'm 👍 on the changes.

**Comment by yo-bj** [comment]
I think it's safe to pull this one in - I'll ping the LPC channel in Slack to give them a heads up on the changes, but any changes should be done by next week (month before the conference).

**Comment by yo-bj** [comment]
Posted to both the listserv and the LPC channel on Slack last week/early this week, and no comments either way. Since there are no comments, an Owner can merge away.

**Pull request closed without merging**